chain/chunks/src/shards_manager_actor.rs

```diff
@@ -169,7 +169,11 @@ pub enum ProcessPartialEncodedChunkResult {
     NeedMorePartsOrReceipts,
     /// PartialEncodedChunkMessage is received earlier than Block for the same height.
     /// Without the block we cannot restore the epoch and save encoded chunk data.
+    /// The chunk is partially processed.
     NeedBlock,
+    /// PartialEncodedChunkMessage is received earlier than Block for the same height.
+    /// The chunk has been dropped without processing any part of it.
+    NeedBlockChunkDropped,
```

Comment:
```suggestion
    NeedsBlockChunkDropped,
```

nit: the grammar is off in the existing variants but we can do better =).